.github/workflows/r-ci.yml

```diff
@@ -19,6 +19,7 @@ env:
 jobs:
   ci:
     strategy:
+      fail-fast: false
```

Comment:
Revert?

Reply:
I would keep it simply because sometimes Coverage fails first and I find it informative to see what what macOS and standard non-Coverage Linux get up to.  Which is why I kept.

But not putting up a fight over it. If you (or someone else) feels the line must go we can delete it.

Reply:
Doesn't matter to me either way

Reply:
I would like to voice favor for keeping `fail-fast: false`; the coverage CI often fails first and swallows errors when it does. Allowing the Linux and macOS CIs to keep running will be beneficial for debugging errors